src/guide/jbs-jdk-bug-system.md

```diff
@@ -219,9 +219,9 @@ Now that the issue is in the right component and has the basic information, the
     * In addition to the version where the bug was found, take special care to also investigate if the bug affects mainline.
       * See [Indicating what releases an issue is applicable to](#indicating-what-releases-an-issue-is-applicable-to) for more details.
     * For enhancements the [Affects Version/s]{.jbs-field} should be empty unless you feel that the enhancement is only relevant to a particular release family, and shouldn't go into a future mainline release.
-    * [Affects Version/s]{.jbs-field} should never use any of the "special" values available in JBS like [tbd]{.jbs-value}, [na]{.jbs-value}, [unknown]{.jbs-value} or similar. Only actual JDK release numbers or [-pool]{.jbs-value} versions should be used.
+    * [Affects Version/s]{.jbs-field} should never use any of the "special" values available in JBS like [tbd]{.jbs-value}, [na]{.jbs-value}, [unknown]{.jbs-value}, [(Rel)-pool]{.jbs-value} or similar. Only actual JDK release numbers versions should be used, if you want to reflect that an issue is relevant to an older release use the family release value [8]{.jbs-value}, [17]{.jbs-value}.
```

Comment:
> Only actual JDK release numbers versions 

should be

> Only actual JDK release numbers

The "versions" came from "or pool versions".

Reply:
fixed